packages/material-components-web/material-components-web.scss

```diff
@@ -33,6 +33,7 @@
 @import "@material/select/mdc-select";
 @import "@material/snackbar/mdc-snackbar";
 @import "@material/switch/mdc-switch";
+@import "@material/tabs/mdc-tabs";
```

Comment:
Yes. It didn't clear this comment because the comment is in `material-components-web.scss`